advanced/kernel-messaging/src/index.ts

```diff
@@ -27,27 +29,30 @@ const extension: JupyterFrontEndPlugin<void> = {
   id: 'kernel-messaging',
   autoStart: true,
   optional: [ILauncher],
-  requires: [ICommandPalette, IMainMenu],
+  requires: [ICommandPalette, IMainMenu, ITranslator],
   activate: activate
 };
 
 /**
  * Activate the JupyterLab extension.
  *
- * @param app Jupyter Font End
+ * @param app Jupyter Front End
  * @param palette Jupyter Commands Palette
  * @param mainMenu Jupyter Menu
  * @param launcher [optional] Jupyter Launcher
+ * @param translator Jupyter Translator
  */
 function activate(
   app: JupyterFrontEnd,
   palette: ICommandPalette,
   mainMenu: IMainMenu,
-  launcher: ILauncher | null
+  launcher: ILauncher | null,
+  translator: ITranslator
```

Comment:
Maybe we could switch this 2 to make it more readable and have "Optional" items at the end?

What do you think @kgoo124 ?

Reply:
Yes, that works! Fixed